### DISCUSSION TITLE
How to not make an image not repeat itself in a puzzle?

### DISCUSSION THREAD

**Discussion by singlesammy1996** [flbulgarelli/headbreaker#69] [Q&A]
I am trying to make a puzzle with a custom image, so I make every piece the size with the same ratio the image has.
For example, the original image is 1000 x 500, I make a 10x10 puzzle, and then every piece is 100x50. As calculated the whole puzzle would be just the size as the original image, right? But instead I got a reapeating image, like this.
![屏幕截图 2023-08-26 154335](https://github.com/flbulgarelli/headbreaker/assets/77719460/df02fa2b-723a-41b8-9820-f5876f0518ca)
the upper is the original pic
the latter is the puzzle, same size but repeating
I'd rather get an image that is cut off a bit than it repeating itself.
Not sure what to do with this

**Comment by singlesammy1996** [comment]
I changed the "puzzleDiameter" property, it helps, but still don't know what this property means, so I can't change this accordingly

**Reply by lucawittorf** [reply]
Hi @singlesammy1996 , how exactly did you use the puzzlediameter value ? I have the same problem but dont know how use it.

**Reply by singlesammy1996** [reply]
I'm not sure either, but I set it like this: 
`{x: imageWidth + a, y: imageHeight + a}`
"a" is a regular constant, I just randomly give it 26
